index.html

```diff
@@ -0,0 +1,368 @@
+
+![](images/multi-monitor-setup.png)
+*<br>An example [=multi-monitor setup=] optimized for software development tasks with a [=traditional laptop=] connected to two [=monitors=] with one monitor in landscape orientation displaying code review side by side and one monitor in portrait orientation displaying a long change log listing. ([Source](TODO))*
+
+### Monitor
+
+A <dfn>monitor</dfn> is an output device that displays information in pictorial form. A monitor usually comprises the display, circuitry, casing, and power supply. ([Wikipedia](https://en.wikipedia.org/wiki/Computer_monitor))
+
+### Display
+
+A <dfn>display</dfn>, informally a [=screen=], is a display device for presentation of images, text, or video transmitted electronically, without producing a permanent record. Electronic visual displays include television sets, computer monitors, and digital signage. ([Wikipedia](https://en.wikipedia.org/wiki/Electronic_visual_display))
+
+### Screen
+
+<dfn>Screen</dfn> is a synonym to [=display=]. [=Screen=] is used more in software context (logical) while display is primarily used in hardware context (physical). See also the [=Screen interface=].
+
+### Multiple internal display
+
+A <dfn>multiple internal display</dfn> (MID) device is a more technical synonym to a [=dual-screen device=].
+
+### Single foldable display
+
+A <dfn>single foldable display</dfn> (SFD) refers to a [=foldable-screen device=] with a single screen.
+
+### Posture
+
+A <dfn>posture</dfn> is defined as the primary way in which the [=foldable device=] is positioned and presented to the user. Because [=foldable devices=] have a hinge they can present different [=postures=] depending on the hinge angle, device orientation, and the way on how the device is being held by the user. Example postures include “book”, “flat”, “laptop”, and “tent”. The list of postures is expected to evolve over time, and not all foldable devices are able to support all possible postures.
+
+![](images/postures.png)
+*<br>Example postures from left to right: “closed”, “laptop”, “table top”, “flat”, “canvas”, and “book”. ([Source](TODO))*
+
+### Spanning
+
+<dfn>Spanning</dfn> is a concept that allows apps or websites to run side by side on a [=dual-screen device=]. The ability to maximize a given application across the two [=screens=] is enabled through the [=window manager=]. Because spanning across the two screens double the screen estate it is expected that applications and websites leverage that additional space to optimize or enhance the UX to provide a better experience.
+
+### Window segments
+
+A <dfn>window segment</dfn> represents the regions (and their dimensions) of the window that reside on separate (adjacent) [=displays=].
```

Comment:
With something like FancyZones this doesn't necessarily need to be separate physically displays

Reply:
Can we have a picture example?

Reply:
OK.

Reply:
@arskama's feedback suggests we need to improve the definitions of "display" and "screen". These terms are overloaded and used differently in different contexts in general (e.g. software vs. hardware, formal vs. informal use). I believe we should define these terms in a way they make sense to use in the context of the specs in scope of this explainer.

@arskama, can you investigate this? Maybe we this could be an issue of its own. Feel free to open one.

Reply:
@anssiko ok, I ll suggest something in context of these specs.